package.json

```diff
@@ -27,7 +28,12 @@
     "babel-plugin-transform-object-rest-spread": "^6.23.0",
     "babel-preset-es2015": "^6.18.0",
     "babel-register": "^6.18.0",
-    "eslint": "^3.12.2",
+    "eslines": "^0.0.12",
+    "eslint": "^3.13.0",
+    "eslint-eslines": "0.0.1",
```

Comment:
Thanks for informing me about this possibility :)
However, I personally found pre-commit warnings quite valuable. Could we leave it as it is for now and update it once we can keep or configure the warnings?